examples/human_in_the_loop.py

```diff
@@ -1,7 +1,17 @@
+"""
+This examples runs the human-keyboard Agent, which allows you to control and monitor input devices.
+
+NOTE: You will need to install [extras] to run this example. `pip install -e .[extras]`
+"""
+
```

Comment:
Seeing this now, I wonder if it would be best to remove the `human_in_the_loop` example and move the agent to the zoo. Then this requirement can sit in the agent's requirements.

I do not think we need a completely separate example to demonstrate an agent that would generally be used for debugging.

Reply:
I think this is fine for now but please add an issue to move the `HumanKeyboardAgent` to the agent zoo.

Reply:
@qianyi-sun Can you please follow up on these changes?